src/widgets/splits/Split.cpp

```diff
@@ -636,6 +636,16 @@ void Split::showViewerList()
     viewerDock->activateWindow();
 }
 
+void Split::openSubPage()
```

Comment:
I think you should remove the first commit that merges your other branch.

Reply:
True, I only merged that so I can use it in my own compiled version anyway. Not related to this feature.

Reply:
In my opinion, it would be better to really drop the first (d3aa3694f2d305b6f38f71809efb5dfc87b51061) and last (18349ed9b3f35ccb59c658a04e2c71b6817c08e3) commits to make the history of changes look a little bit nicer.
One of the many ways to do this is to use `git rebase -i HEAD~25`, manually delete the first and last lines and save the result and then use `git push --force`.
(Or just use any gui git client.)